reddit/data.go

```diff
@@ -35,6 +37,12 @@ type Comment struct {
 	Distinguished string `mapstructure:"distinguished"`
 }
 
+// IsRoot is true when the comment is a top level comment.
+func (c *Comment) IsRoot() bool {
```

Comment:
Based on this comment I think IsTopLevel would be a more appropriate name; more Reddit users could intuit what that means.